Modules/Core/Common/include/itkConceptChecking.h

```diff
@@ -852,7 +852,7 @@ template <typename T>
 struct IsInteger
 {
   using Self = IsInteger;
-  static constexpr bool Integral = NumericTraits<T>::IsInteger;
+  static constexpr bool Integral = std::is_integral<T>::value;
```

Comment:
Commit message's title should be changed to `STYLE: Remove uses of IsInteger and replace with std::is_integral`

Reply:
Fixed.